generators/app/templates/src/containers/TicTacToe/GameListPage/children/Form/game-form.component.js

```diff
@@ -0,0 +1,237 @@
+            const predicate = `${prefix}${field.predicate}`;
+            await document[predicate].add(setupObj[field.predicate]);
+          }
+          /**
+           * Find opponent game inbox if doesn't exist get global
+           * @to: Opponent inbox path
+           */
+          const to = helperNotification.getDefaultInbox(inboxes, 'Game', 'Global');
+          const target = `${window.location.href}/${btoa(documentUri)}`;
+          await sendNotification(
+            {
+              title: 'Tictactoe invitation',
+              summary: 'has invited you to play Tic-Tac-Toe.',
+              actor: webId,
+              object: documentUri,
+              target
+            },
+            to.path
+          );
+
+          setDocumentUri(`${Date.now()}.ttl`);
+        } else {
+          errorToaster(`${opponent} ${t('game.createFolder')}`, 'Error');
+        }
+      } else {
+        errorToaster(`${opponent} ${t('noInboxOpponent.message')}`, 'Error', {
+          label: t('noInboxOpponent.link.label'),
+          href: t('noInboxOpponent.link.href')
+        });
+      }
+    } catch (e) {
+      throw new Error(e);
+    }
+  };
+
+  const createACLFile = async (documentUri, body) => {
+    try {
+      ldflexHelper.createDocumentWithTurtle(documentUri, body);
+    } catch (e) {
+      throw new Error('Error while creating ACL');
+    }
+  };
+
+  const createQuad = (subject, predicate, object) => ({
+    subject,
+    predicate,
+    object
+  });
+
+  const createDefaultQuads = (modes, webId, subject, prefixes) => {
+    const { acl, foaf, a } = prefixes;
+    const originalPredicates = [
+      createQuad(subject, `${a}`, namedNode(`${acl}Authorization`)),
+      createQuad(subject, `${acl}accessTo`, namedNode(documentUri))
+    ];
+    const predicates = webId
+      ? [...originalPredicates, createQuad(subject, `${acl}agent`, namedNode(webId))]
+      : [...originalPredicates, createQuad(subject, `${acl}agentClass`, namedNode(`${foaf}Agent`))];
+
+    const quadArray = modes.reduce(
+      (array, mode) => [...array, createQuad(subject, `${acl}mode`, namedNode(`${acl}${mode}`))],
+      predicates
+    );
+
+    return quadArray;
+  };
+
+  const aclTurtle = async (documentUri, opponent) => {
+    try {
+      const aclDocument = `${documentUri}.acl`;
+      const { DataFactory } = N3;
+      const { namedNode, quad } = DataFactory;
+      const prefixes = {
+        acl: 'http://www.w3.org/ns/auth/acl#',
+        foaf: 'http://xmlns.com/foaf/0.1/',
+        n: 'http://www.w3.org/2006/vcard/ns#',
+        '': `${aclDocument}#`,
+        a: 'http://www.w3.org/ns/auth/acl#type'
+      };
+      const owner = `${aclDocument}#owner`;
+      const publicSubject = `${aclDocument}#public`;
+      const opponentSubject = `${aclDocument}#opponent`;
+
+      const writer = new N3.Writer({ prefixes });
+      const ownerQuads = createDefaultQuads(['Control', 'Read', 'Write'], webId, owner, prefixes);
```

Comment:
Permission names should be in an object or enum instead of "magic strings"

Reply:
This will be changed once the new ACL library gets added